lib/mix/tasks/absinthe.schema.json.ex

```diff
@@ -100,4 +119,20 @@ defmodule Mix.Tasks.Absinthe.Schema.Json do
         [value] |> Module.safe_concat()
     end
   end
+
+  defp write_schema(content, filename) do
+    create_directory(Path.dirname(filename))
+    create_file(filename, content, force: true)
+  end
+
+  defp check_function_available(module, func) do
```

Comment:
This basically just recreates the Elixir standard library error right?

```
iex(1)> Foo.encode!
** (UndefinedFunctionError) function Foo.encode!/0 is undefined (module Foo is not available)
    Foo.encode!()
```

I think we should just call encode! and if it doesn't work let the default error mesage do its thing.

Reply:
Yeah, that makes sense. I can't remember what led me down that path. ;-)